src/odemis/util/conversion.py

```diff
@@ -104,9 +104,12 @@ def rgb_to_frgb(rgb):
     :return: (float, float, float)
 
     """
-
+    # check if the tuple consists of three values
     if len(rgb) != 3:
         raise ValueError("Illegal RGB colour %s" % rgb)
+    # check if the tuple values are valid RGB values
+    if not numpy.all(tuple(0 <= c <= 255 for c in rgb)):
+        raise ValueError("Not all values are valid RGB values (0-255)")
```

Comment:
Please include the actual value of `rgb` in the message (to make the debugging easier).